pavelib/assets.py

```diff
@@ -192,31 +200,37 @@ def compile_sass(options):
     Compile Sass to CSS.
     """
     debug = options.get('debug')
-    parts = ["sass"]
-    parts.append("--update")
-    parts.append("--cache-location {cache}".format(cache=SASS_CACHE_PATH))
-    parts.append("--default-encoding utf-8")
+
     if debug:
-        parts.append("--sourcemap")
+        source_comments = True
```

Comment:
Sorry if I'm missing something, but I don't see this line or the next one being used...are they?